frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -163,11 +161,6 @@ export const ParametersAndCardsContainer = styled.div<{
     ${CardsContainer} {
       padding-bottom: 20px;
     }
-
-    ${DashCard.root} {
```

Comment:
This code is inside the `ParametersAndCardsContainer` component which is not used in the public or embedded dashboard so I moved the code around